hermes/probe/probe.go

```diff
@@ -38,6 +38,33 @@ import (
 	journalpb "github.com/googleinterns/step224-2020/hermes/proto"
 )
 
+// ProbeError is an error that includes an exit status for used within Hermes.
+type ProbeError struct {
+	// TODO(evanSpendlove): Refactor metrics.ExitStatus to be named something similar to metrics.APICallStatus.
+	Status metrics.ExitStatus
+	Err    error
+}
+
+// NewProbeError returns a new ProbeError containing the error and status passed.
+// Arguments:
+//	- status: pass the exit status associated with this error.
+//	- err: the underlying error.
+// Returns:
+//	- ProbeError: returns a new ProbeError object containing the args passed.
+func NewProbeError(status metrics.ExitStatus, err error) *ProbeError {
```

Comment:
This function doesn't do anything.
Just remove it and use &ProbeError{...} in its place.